recipes/tbot/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "tbot" %}
+{% set version = "0.8.2" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/Rahix/tbot/archive/v0.8.2.tar.gz
```

Comment:
This should be templated in order to receive  automatic updates: `https://github.com/Rahix/tbot/archive/v{{ version }}.tar.gz`

Reply:
Oh Yes. Sorry I missed it